vcs-integration/git.sh

```diff
@@ -63,7 +64,7 @@ _git_increment_ () {
 }
 
 _git_commitdate_ () {
-    REPLY="$(date -d @"$(git show -s --format=%ct $1)" '+%Y-%m-%d')"
+    REPLY="$(date -d @"$(git show -s --format=%ct)" '+%Y-%m-%d')"
```

Comment:
There is some inconsistency in the library. Probably, it would be best to use an argument for all these types of functions. Maybe do `committish="${1:-HEAD}"` (when it makes sense), which is what `git` often does internally.

It's true, that it is not used that way at the moment, but it improves future utility at virtually zero cost.